extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -677,6 +679,23 @@ function getClusterContexts(file: string): (() => Promise<OptionsInfo>) {
 	};
 }
 
+async function throwIfNotExistsOrNotAFile(file: string, fileDescription: string = 'kube config') {
+	try {
+		const stats = await fs.promises.stat(file); // this throws if the file does not exist with error.code = ENOENT
+		if (!stats.isFile()) {
+			throw Error(localize('getClusterContexts.NotFile', "Path:{0} is not a file, please select a valid {1} file ...", file, fileDescription));
+		}
+	}
+	catch (e) {
+		if (e.code === 'ENOENT') {
+			throw Error(localize('getClusterContexts.FileNotFound', "File:{0} not found. Please select a {1} file ...", file, fileDescription));
```

Comment:
I'd get rid of the `...` too, I'm not a fan of the "dangling" sentences like that for UI elements.

Reply:
fixed